app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -1614,16 +1615,77 @@ class ChatActivity :
     }
 
     private fun startPlayback(message: ChatMessage, doPlay: Boolean = true) {
-        if (!active) {
+        //if (!active) {
             // don't begin to play voice message if screen is not visible anymore.
             // this situation might happen if file is downloading but user already left the chatview.
             // If user returns to chatview, the old chatview instance is not attached anymore
             // and he has to click the play button again (which is considered to be okay)
-            return
-        }
+        //    return
+        //}
 
         initMediaPlayer(message)
 
+        val id = message.id.toString()
+        val index = adapter?.getMessagePositionById(id) ?: 0
+
+        var nextMessage : ChatMessage? = null
+        for (i in -5..5) {
+            if(index - i < 0){
+                break
+            }
+            if(i == 0 || index - i >= (adapter?.items?.size?: 0) ){
+                continue
+            }
+            val curMsg = adapter?.items?.getOrNull(index - i)?.item
+            if(curMsg is ChatMessage) {
+                if(nextMessage == null && i > 0) {
+                    nextMessage = curMsg as ChatMessage
+                }
+
+                if(curMsg.isVoiceMessage){
+                    if(curMsg.selectedIndividualHashMap == null){
+                        // WORKAROUND TO FETCH FILE INFO:
+                        curMsg.getImageUrl()
```

Comment:
yes indeed this workaround is necessary here :+1:  The getImageUrl() method is really confusing and needs to be changed in the future!